File: Array/Rotate_Array.js
let rotate = function(nums, k) {
    if (nums.length == 1) return
    let K = k >= nums.length ? k-nums.length*(Math.floor(k/nums.length)) : k
    let pivot = nums.length-K-1
    let back = nums.splice(0,pivot+1)
    nums.push(...back)
    return nums
};

File: Array_jeongyeon/RotateArray.py
class Solution(object):
    def rotate(self, nums, k):
        copy_nums = list(nums)
        for i in range(0,len(nums)):
            if i+k >= len(nums):
                j = i-(k+1)
                nums[j] = copy_nums[i]
            else:
                j = i+k
                nums[j] = copy_nums[i]